vagrant/Vagrantfile

```diff
@@ -790,20 +790,21 @@ Vagrant.configure('2') do |config|
 
       configure_docker_network c
 
-      configure_lustre_network c
-
-      c.vm.provision 'install-iml-local',
-            type: 'shell',
-            run: 'never',
-            path: './scripts/install_iml_local_agent.sh'
-
       c.vm.provision 'install-lustre-client',
                      type: 'shell',
                      run: 'never',
                      inline: <<-SHELL
                             yum-config-manager --add-repo https://downloads.whamcloud.com/public/lustre/lustre-#{LUSTRE}/el7/client/
                             yum install -y --nogpgcheck lustre-client
                      SHELL
+
+      configure_lustre_network c
+
+      c.vm.provision 'install-iml-local',
+            type: 'shell',
+            run: 'never',
+            path: './scripts/install_iml_local_agent.sh'
```

Comment:
This shouldn't need to be moved.

Reply:
It is not :) This is how Github shows the changes/

Reply:
I mean before it was

1. provision LNet
1. Install IML local
1. Install Lustre client RPMs

Now it is:

1. Install Lustre Client
1. provision LNet
1. Install IML local

The order it should probably be in is:

1. Install IML local
1. Install Lustre Client
1. provision LNet

Reply:
Ah, ok. I just tried a minimal change. Btw, is there a reason to install local iml on clients?